providers/flagd/pom.xml

```diff
@@ -121,12 +145,12 @@
                                 <argument>submodule</argument>
                                 <argument>update</argument>
                                 <argument>--init</argument>
-                                <argument>schemas</argument>
+                                <argument>--remote</argument>
```

Comment:
This change makes the build non-deterministic. Every time it runs it will attempt to update the submodule, regardless of the previously set tag.

See https://git-scm.com/book/en/v2/Git-Tools-Submodules

Reply:
yeah, this is true. I had trouble fetching the latest schema updates and used this to force an update. I will try to fix this

Reply:
Fixed and reverted :)